Update README.md. Add spanSize, scrollOffset, refreshVisibleItem and isItemAtIndexVisible

vallemar opened the pull request.

farfromrefug commented:
@vallemar thanks a lot for this. Could you just add the fact that it is based on `rowHeight`  and not `colWidth` when horizontal?

Thanks again!

vallemar commented:
@farfromrefug I have updated the spanSize text with the reference of vertical and horizontal and also added scrollOffset, refreshVisibleItem and isItemAtIndexVisible

farfromrefug commented:
@vallemar awesome!
